src/main/kotlin/com/makarytskyi/rentcar/repository/impl/SimpleCarRepository.kt

```diff
@@ -0,0 +1,44 @@
+package com.makarytskyi.rentcar.repository.impl
+
+import com.makarytskyi.rentcar.model.Car
+import com.makarytskyi.rentcar.repository.CarRepository
+import org.bson.types.ObjectId
+import org.springframework.stereotype.Repository
+
+@Repository
+class SimpleCarRepository: CarRepository {
+    val map: MutableMap<String, Car> = HashMap()
+
+    override fun findById(id: String): Car? = map[id]
+
+    override fun save(car: Car): Car {
+        val id = ObjectId().toString()
```

Comment:
```suggestion
        val id = ObjectId().toHexString()
```
usually hex representation of ObjectId is used